presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java

```diff
@@ -91,334 +90,304 @@ public Requirement getRequirements(Configuration configuration)
     @Requires(ImmutableAllTypesTable.class)
     public void preparedSelectApi()
     {
-        if (usingTeradataJdbcDriver(connection())) {
-            String selectSql = "SELECT c_int FROM " + TABLE_NAME + " WHERE c_int = ?";
-            final int testValue = 2147483647;
+        String selectSql = "SELECT c_int FROM " + TABLE_NAME + " WHERE c_int = ?";
+        int testValue = 2147483647;
 
-            assertThat(query(selectSql, param(INTEGER, testValue))).containsOnly(row(testValue));
+        assertThat(query(selectSql, param(INTEGER, testValue))).containsOnly(row(testValue));
 
-            assertThat(query(selectSql, param(INTEGER, null))).hasNoRows();
+        assertThat(query(selectSql, param(INTEGER, null))).hasNoRows();
 
-            assertThat(query(selectSql, param(INTEGER, 2))).hasNoRows();
-        }
-        else {
-            LOGGER.warn("preparedSelectApi() only applies to TeradataJdbcDriver");
-        }
+        assertThat(query(selectSql, param(INTEGER, 2))).hasNoRows();
     }
 
     @Test(groups = {JDBC, SIMBA_JDBC})
     @Requires(ImmutableAllTypesTable.class)
     public void preparedSelectSql()
             throws SQLException
     {
-        if (usingTeradataJdbcDriver(connection())) {
-            String prepareSql = "PREPARE ps1 from SELECT c_int FROM " + TABLE_NAME + " WHERE c_int = ?";
-            final int testValue = 2147483647;
-            String executeSql = "EXECUTE ps1 using ";
+        String prepareSql = "PREPARE ps1 from SELECT c_int FROM " + TABLE_NAME + " WHERE c_int = ?";
+        int testValue = 2147483647;
+        String executeSql = "EXECUTE ps1 using ";
 
-            Statement statement = connection().createStatement();
-            statement.execute(prepareSql);
+        Statement statement = connection().createStatement();
+        statement.execute(prepareSql);
 
-            assertThat(QueryResult.forResultSet(statement.executeQuery(executeSql + testValue)))
-                    .containsOnly(row(testValue));
-            assertThat(QueryResult.forResultSet(statement.executeQuery(executeSql + "NULL")))
-                    .hasNoRows();
-            assertThat(QueryResult.forResultSet(statement.executeQuery(executeSql + 2)))
-                    .hasNoRows();
-        }
-        else {
-            LOGGER.warn("preparedSelectSql() only applies to TeradataJdbcDriver");
-        }
+        assertThat(QueryResult.forResultSet(statement.executeQuery(executeSql + testValue)))
+                .containsOnly(row(testValue));
+        assertThat(QueryResult.forResultSet(statement.executeQuery(executeSql + "NULL")))
+                .hasNoRows();
+        assertThat(QueryResult.forResultSet(statement.executeQuery(executeSql + 2)))
+                .hasNoRows();
     }
 
     @Test(groups = {JDBC, SIMBA_JDBC})
```

Comment:
I think this tests no longer needs to be in both groups. I would remove `SIMBA_JDBC` and keep only `JDBC`.

Their use case is to run tempto with all JDBC tests but these that are SIMBA_JDBC (or PRESTO_JDBC depending on used driver). As I understand here we expect both drivers to be compatible.

Reply:
@kokosing when testing simba, do we run sth like `-g JDBC -x PRESTO_JDBC` or `-g SIMBA_JDBC` ?

Reply:
`-g ldap,jdbc,simba_jdbc -x presto_jdbc,mysql,quarantine`

Reply:
or
`-g smoke,simba_jdbc,jdbc -x presto_jdbc,ldap,mysql,quarantine`

Generally rule is the same